tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -134,6 +137,11 @@ def tap_back_from_import_contacts(self):
         header.tap(25, 25)
```

Comment:
Hrm, this is actually something that could be fixed, since we're now able to use shadow dom. I think it's a simple matter of `GaiaHeader(self.marionette).go_back()`, but I guess that's not in the scope of this pull request.